changelogs/current.yaml

```diff
@@ -86,6 +86,9 @@ new_features:
 - area: generic_proxy
   change: |
     added :ref:`generic rds support <envoy_v3_api_field_extensions.filters.network.generic_proxy.v3.GenericProxy.generic_rds>`.
+- area: listener
+  change: |
+    added ``continueFilterChain()`` and ``dispatcher()`` methods to the ``ListenerFilterCallback``, enable the listener filter back to the lister filter iteration after some async process done.
```

Comment:
maybe instead of :`, enable the listener filter back to the lister filter iteration after some async process done.`

something like :`. This allows listener filters to continue listener filter iteration after stopping iteration e.g. if the listener filter depends on an async process.`

Reply:
thanks! done